osu.Framework/Graphics/Textures/Texture.cs

```
@@ -183,7 +183,9 @@ public RectangleF GetTextureRect(RectangleF? textureRect = null)
 
         public void Draw(Quad vertexQuad, Color4 colour, RectangleF? textureRect = null, VertexBatch<TexturedVertex2D> spriteBatch = null)
         {
-            TextureGL?.Draw(vertexQuad, textureBounds(textureRect), colour, spriteBatch);
+            if (TextureGL == null || !TextureGL.Bind()) return;

```

Do we want to silently return, or do we want to Assert? Silent returns make debugging a lot harder than failing loudly.